src/cpp/database/database.cpp

```diff
@@ -13,13 +13,13 @@
  * limitations under the License.
  */
 
-#include "database.hpp"
+#include <iostream>
+#include <mutex>  // For std::unique_lock
 
 #include <fastdds-statistics-backend/exception/Exception.hpp>
 #include <fastdds-statistics-backend/types/types.hpp>
 
-#include <mutex>  // For std::unique_lock
-#include <iostream>
+#include "database.hpp"
```

Comment:
Exception to the header order: The header corresponding to the current source file must be the first one